backend/routes.js

```diff
@@ -30,6 +33,13 @@ const configureRoutes = ({ oauthApi, manageUsersApi, nomisUsersAndRolesApi }) =>
   router.use('/manage-dps-users/:userId', manageDpsUserRouter({ oauthApi, nomisUsersAndRolesApi, manageUsersApi }))
   router.use('/manage-groups', manageGroupsRouter({ oauthApi, manageUsersApi }))
   router.use('/manage-roles', manageRolesRouter({ oauthApi, manageUsersApi }))
+  router.use('/email-domains', manageEmailDomainsRouter({ oauthApi, manageUsersApi }))
+  router.use('/create-email-domain', createEmailDomainsRouter({ oauthApi, manageUsersApi }))
+  router.use(
+    '/delete-email-domain?id=:domainId&name=:domainName',
+    deleteEmailDomainsRouter({ oauthApi, manageUsersApi }),
+  )
+  router.use('/delete-email-domain', deleteEmailDomainsRouter({ oauthApi, manageUsersApi }))
```

Comment:
Do these need oauthApi?

Reply:
As discussed, pattern predates this PR, but changes have been done retrospectively.